### ISSUE TITLE
Option to add daily forecasts to main screen

### ISSUE THREAD

**Issue by bb1a2** [thuryn/your-local-weather#71]
90% of the time, I don't need sunrise and sunset, but I need tomorrow's temperatures!
The current lower half  of the screen can be comfortably placed under the Now temperature.

I suggest an option to replace the lower half of the screen with a daily forecast consisting of: high and low temperatures and weather condition icon.

Example:
![a](https://user-images.githubusercontent.com/38409206/44203479-5c6dfb00-a14f-11e8-8712-dd26bea5c526.png)

Also, I think daily forecast needs an option to show fewer temperatures (just high and low). Too much info can be confusing. This is an another example of just the right amount in my opinion:

![b](https://user-images.githubusercontent.com/38409206/44203480-5c6dfb00-a14f-11e8-8690-1d4a58ff5bfd.png)

**Thank you**

**Comment by neomodern** [comment]
Agreed. All I usually need to know is the high and low temperatures for each day and the weather conditions icon. 

My ideal view would use the previous Good Weather app's Daily Forecast view, but modify the more info box for each day (which shows when clicked). This would instead use the configurable column data from your fork to show a full breakdown of the 8 3-hour readings. So basically a combination of the views from the previous app and yours.

**Comment by dimqua** [comment]
In my opinion, hourly forecast in main screen would be nice to have too.

**Comment by dimqua** [comment]
This is how it can looks like:

![](https://i.imgur.com/Encdz5y.png)

(taken from [Privacy Friendly Weather](https://github.com/SecUSo/privacy-friendly-weather) app)
